weldx/transformations/local_cs.py

```diff
@@ -450,9 +450,9 @@ def from_axis_vectors(
         y: types_coordinates = None,
         z: types_coordinates = None,
         coordinates: types_coordinates = None,
-        time: types_time_like = None,
-        time_ref: types_timestamp_like = None,
-    ):
+        time: Union[types_time_like, Time] = None,
+        time_ref: Union[types_timestamp_like, Time] = None,
+    ) -> LocalCoordinateSystem:
```

Comment:
`Time` should be a part of those definitions. Looked into the definitions and saw, that I somehow missed putting it into `types_time_like` but it is already in types_timestamp_like. See here: 
https://github.com/BAMWelDX/weldx/blob/8a40edb2b3e7469c79e083636e25fcd45df5ac9f/weldx/time.py#L675

Reply:
Just realized, that `Time` should already be included in `types_time_like` since it is composed of other type collections

Reply:
oh I see, thanks
this somehow looked weird after merging master into this branch